packages/api/amplify_api/lib/amplify_api.dart

```diff
@@ -17,6 +17,7 @@ library amplify_api_plugin;
 
 import 'dart:io';
 
+import 'package:amplify_api/src/api_plugin_impl.dart';
 import 'package:amplify_api/src/method_channel_api.dart';
```

Comment:
nit: method channel import not needed anymore so please remove

Later (not in this PR) should remove related code.